kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -11,36 +11,71 @@
     text-overflow: ellipsis;
 }
 
+.kb-data-breadcrumb a {
```

Comment:
Do you know if Andy and Mallory want all anchor elements and the bread crumbs to have this cursor?

Reply:
@eamahanna The breadcrumbs are not part of this ticket, the only change is moving from inline to a css sheet. My guess is yes, it's not fun to click an html link when your cursor doesn't change into a cursor that you get when you hover a link